mayastor/src/bdev/dev/nvmx/controller.rs

```diff
@@ -96,22 +106,35 @@ pub struct NvmeControllerInner<'a> {
     adminq_poller: poller::Poller<'a>,
     io_device: Arc<IoDevice>,
 }
+
+type EventCallbackList = Vec<fn(DeviceEventType, &str)>;
```

Comment:
Should we limit the number of entries we can insert? (or maybe a different data structure) or are the number of events, not a concern? If they *might* be, I think we should add an issue for it to look into it later.

Reply:
Sure thing. I will create a ticket to consider improvement in device events storing.